spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/thymeleaf/ThymeleafProperties.java

```diff
@@ -97,6 +100,11 @@
 	private String[] excludedViewNames;
 
 	/**
+	 * Template to use for the root view of an application.
+	 */
+	private String rootView = DEFAULT_ROOT_VIEW;
```

Comment:
Does this need to be configurable? We require a static welcome page to be called `index.html`. I wonder if we could do the same for a templated welcome page. It may clash with an existing `index` template but I'd like to explore the possibility of somehow not requiring a configuration option.